components/pkg-export-kubernetes/src/manifest.rs

```diff
@@ -105,3 +136,62 @@ impl Manifest {
         Ok(())
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_manifest_generation() {
+        let mut m = Manifest {
+            metadata_name: "nginx-f84iq".to_owned(),
+            habitat_name: "nginx".to_owned(),
+            image: "core/nginx".to_owned(),
```

Comment:
We still don't have the tag?

Reply:
but the "latest" tag is implicit here. I thought I'll only add the tag when we have version (and maybe release) in the service name and I think we agreed to do that in a separate PR after this.

Reply:
Yes, separate PR is fine for this. I still like to do `latest` by default rather then omitting the tag.